src/mapper/cmd/main.go

```diff
@@ -89,6 +93,30 @@ func main() {
 		os.Exit(1)
 	}
 
+	// create webhook server certificate
+	logrus.Infoln("Creating self signing certs")
```

Comment:
This entire section for the webhook should be conditional on AWS visibility being enabled. Otherwise, it will fail when it's not, since the webhook config doesn't exist.
It should also be disabled by default for local run.